body/stretch_body/wacc.py

```diff
@@ -215,30 +179,110 @@ def pack_config(self,s,sidx):
             sidx += 1
             return sidx
 
+    def unpack_status(self,s):
+        pass
```

Comment:
Would be better to `raise NotImplementedError()` here.

Reply:
Done